### ISSUE TITLE
Your Feedback for the Submit News form.

### ISSUE THREAD

**Issue by CaMer0n** [e107inc/e107#1796]
Submit news needs a revamp. It is missing some important fields such as 'summary' and 'keywords/tags'. 
What else would you like to see included on the submit news form? @Moc @rica-carv @Jimmi08 @tgtje @lonalore  @MikeyGMT

**Comment by Jimmi08** [comment]
I would like to set in preferences if there fields are editable on frontend. But I really want to see how their editing should be managed by new way on frontend (without admin UI) So my answer is yes.

**Comment by tgtje** [comment]
Well, first one has to think at the general use :
Submitting news (or important things one would like to share).
Now it depends on what the submitter gets in front of him > plain text field (bbcode) or full tiny.
It are options withheld or approved to use. (security takes a peek).

As submitting user i would like to share text of course (textfield), maybe a picture, or a audio file or movie (most prob. very easy).
Now image is present in diff. scenarios, but not the audio/visual part ( neither in upload since that is bound to img).(hence the textfield editoring).

In short : ideas enough, but it is not clear what defines a submitted news item (dependencies to what are to broad in use).
What will one accept and what is needed for displaying inSite.. (and wanted, filtered or not etc..).

A summary field : yes, more upload privileges > yes (strict secur!)
Submit form field for link/url (or all handled by editor ? ) 
So hard to say what exactly (or really needed)..
( i do think theres more in a mix with event calendar options).
Will wait for more reactions....

**Comment by rica-carv** [comment]
I can't see why a user submited news fields can't be as the admin one.
Of course, only admin fields should be rulled out, but almost any content oriented field should be allowed for the standard user to be edited. Logically all this should be reviewed by admin after the news submission by the user....

**Comment by CaMer0n** [comment]
So far I have this to add to the table: 

 submitnews_keywords  varchar(255) NOT NULL default '',  (tags)
 submitnews_description text NOT NULL, (SEO)
 submitnews_summary text NOT NULL, 
 submitnews_media text NOT NULL,   (array containing image, video or audio urls)

**Comment by Jimmi08** [comment]
There is not need for new issue, it's related.
For error and success is used the same string
$mes->addError(LAN_134);
$mes->addSuccess(LAN_134);

Result:
![image](https://github.com/e107inc/e107/assets/5429548/1591c243-f3d0-43a1-8e9c-b36775c50cf6)

Item wasn't submitted because error. A little confusing :)
